backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -134,7 +134,7 @@ trait StandardAsyncExecutionActor extends AsyncBackendJobExecutionActor with Sta
     WomSingleFile(workflowPaths.buildPath(womFile.value).pathAsString)
 
   /** @see [[Command.instantiate]] */
-  final lazy val commandLineValueMapper: WomValue => WomValue = {
+  lazy val commandLineValueMapper: WomValue => WomValue = {
```

Comment:
I remove final keyword because I can not find a more elegant way to map some oss path string in command line, for example: 
task ps {
  String out

  command {
    ps > ${out}
  }

  output {
    File outFile = "${out}"
  }
}

workflow three_step {
  call ps {
    input: out = "oss://bcs-dir/bcs-file"
  }
}